test/test_extension.py

```diff
@@ -20,6 +20,36 @@ def get_config(self, data_dir):
                                             manager_params['database_name'])
         return manager_params, browser_params
 
+    def test_disable_self_id(self, tmpdir):
+        """Verify webdriver self-identification removed from DOM.
+
+        Selenium webdriver self-identifies in two locations in the DOM, see:
+        * https://github.com/SeleniumHQ/selenium/blob/b82512999938d41f6765ce8017284dcabe437d4c/javascript/firefox-driver/extension/content/server.js#L49
+        * https://github.com/SeleniumHQ/selenium/blob/b82512999938d41f6765ce8017284dcabe437d4c/javascript/firefox-driver/extension/content/dommessenger.js#L98
+        """
+
+        from ..automation import CommandSequence
+        def check_for_webdriver(**kwargs):
+            """ Check if webdriver self-identification attributes in the DOM"""
+            driver = kwargs['driver']
+
+            # Check if document element has `webdriver` attribute
+            assert 'true' != driver.execute_script(
```

Comment:
@englehardt these asserts fail when you have `disable_webdriver_self_id=false`, right?
Just wanted to make sure since they are called within a function that is passed around.